models/staging/tpdm/base/base_tpdm__financial_aids.sql

```diff
@@ -0,0 +1,26 @@
+with student_aids as (
+    {{ source_edfi3('student_aids') }}
```

Comment:
If the source object from ed-fi is named student_aids, why is the sql file named financial aids?